src/uniprotkb/components/protein-data-views/__tests__/UniProtKBEntryPublications.spec.tsx

```diff
@@ -19,11 +21,16 @@ describe('UniProtKBEntryPublications', () => {
         <UniProtKBEntryPublications pubmedIds={pubMedIds} />
       </Router>
     );
-    expect(useDataApi).toHaveBeenCalledWith(getPublicationsURL(pubMedIds));
+    expect(useDataApi).toHaveBeenCalledWith(
```

Comment:
Do we need to check that? We're mocking its implementation a few lines above, if we weren't calling it we would get an empty component, and that would affect the snapshot test in the next line.
At most do a check to see if a UI component depending on the mocked data is in the DOM or not (get a link with a pubmed id in the mocked data and check if it's in the document?)